crates/issue/src/lib.rs

```diff
@@ -179,7 +183,7 @@ pub mod pallet {
         Version::V4
     }
 
-    /// Build storage at V1 (requires default 0).
+    /// Build storage at V5 (requires default 4).
```

Comment:
Default is still 4 but actually I'm not sure if we're still using our custom versioning or the pallet version, please check what is set on Kintsugi and Interlay

Reply:
The issue pallet `storageVersion` is set as `4` while the `palletversion` is set as 1 on Interlay as well as Kintsugi. Hence I migrated from `V4` to `V5` `storageVersion`.